src/main/resources/synthetic.xml

```diff
@@ -11,7 +11,6 @@
     <type type="xldeploy.Task" extends="xlrelease.PythonScript" virtual="true">
         <property name="iconLocation" default="xldeploy/xldeploy-icon-black-small.png" hidden="true" />
         <property name="taskColor" hidden="true" default="#68B749" />
-
```

Comment:
Valid point - however the XML wasn't consistently spaced for readability: some types were and some were not. I simply set all types to non-spaced blocks to at least keep it consistent.